### DISCUSSION TITLE
Wyze V3 RTSP stopped HSV

### DISCUSSION THREAD

**Discussion by predator290** [koush/scrypted#401] [Q&A]
I used to have homekit secure video with this camera. Not sure when this stopped but it doesnt seem to be working anymore. I had the RTSP firmware and that stopped working. I switched to the latest firmware and tried using wyze bridge to present a rtsp and that doesnt work either? Am I going crazy? Did something break?

**Comment by CrisB02** [comment]
Same issue here it was working and then it stopped. I haven't tried going back to non-rstp firmware.

**Reply by activex** [reply]
I have a Wyze Cam v3 on the RTSP firmware working in Scrypted and HomeKit.

But I also recently bought another v3, added the RTSP Firmware and added it to scrypted. All settings are the same, but I can’t add the camera to HomeKit. When I scan the QR it times out trying to connect to the device. 

What versions of Scrypted are you all on?

**Reply by MikNolhan** [reply]
> I have a Wyze Cam v3 on the RTSP firmware working in Scrypted and HomeKit.
> 
> 
> 
> But I also recently bought another v3, added the RTSP Firmware and added it to scrypted. All settings are the same, but I can’t add the camera to HomeKit. When I scan the QR it times out trying to connect to the device. 
> 
> 
> 
> What versions of Scrypted are you all on?

Do you see the camera come up if you don't scan and choose to see other devices? I added my V3. It shouldn't be an issue with what camera if it's the same firmware. Try rebooting your router and problem camera. Sounds like routing issue that occurs on some newer more featured routers. Can you add other HomeKit accessories?

**Reply by activex** [reply]
It fixed itself after a few days. Not sure how.
I’d recommend updating scrypted, rebooting the camera and HomeKit hubs.
